### PR TITLE
Add readme.md

### PR DESCRIPTION
**Pull request by Freddie2024**
Don't know if I did this correctly. Maybe readme.md is not a very helpful name of the file. Haven't any ideas what to put inside for now ... Just trying to follow the steps ...

### CODE REVIEW

**Review by cc-co-code — COMMENTED** [review]
nice one

**Review by cc-co-code — COMMENTED** [review]
very creative

File: github-and-markdown-notes.md [hunk 1]
```diff
@@ -1 +1,2 @@
-Don't know what to write in here
```
Comment (cc-co-code) [inline github-and-markdown-notes.md]: why is it red

File: github-and-markdown-notes.md [hunk 2]
```diff
@@ -1 +1,2 @@
...
\ No newline at end of file
+Don't know what to write in here
+Still don't know what to write
```
Comment (cc-co-code) [inline github-and-markdown-notes.md:2]: good

**Review by cc-co-code — COMMENTED** [review]
i dont know what to do

**Comment by ninax1994** [comment]
**nice** 😎

**Pull request merged**